docker-compose.yml

```diff
@@ -87,6 +87,10 @@ services:
       - APPLICANT_OIDC_FIRST_NAME_ATTRIBUTE
       - APPLICANT_OIDC_MIDDLE_NAME_ATTRIBUTE
       - APPLICANT_OIDC_LAST_NAME_ATTRIBUTE
+      - LOGIN_GOV_CLIENT_ID
```

Comment:
Do we really need separate variables for this? Can we use generic applicant oidc variables?

Reply:
+1. Are these equivalent to the `APPLICANT_OIDC_` variables above? It looks like the only new one added here is `ACR`.

`ACR` itself doesn't have strong meaning to me. Do you mind either expanding the acronym or adding a comment somewhere indicating what it stands for?

Reply:
There's a comment in the application.conf.  It's a login.gov specific thing, I don't actually know what it stands for

Reply:
Re: the variable names.  I was thinking about that as well, but I'd rather be consistant with all of them.  I've been toying around with allowing all of the OIDC login providers to just accept the same env variables.

I'm open for discussion, though.

Reply:
I would love all of them to use the same env variables by default. It reduces number of variables we need to write docs for, and removes errors of setting the wrong set of variables.
If overrides are helpful for transition - we can keep that, but hopefully we can get rid of IDCS* variables as well.

Reply:
I also think reusing APPLICANT_OIDC_CLIENT_ID makes sense. Given that it's still OIDC protocol. Amount of variables we need to setup for auth is a bit overwhelming when you do it for the first time. If we want to make sure that correct set of variables is set for given identity provider - we can add that logic to deployment scripts.

Reply:
sounds good, but I'll do this in a different PR. For now, it accepts both